lib/Util/Builtin/GccCompilerSupport.in.cpp

```diff
@@ -23,14 +26,19 @@ class GccCompilerSupport : public Pothos::Util::Compiler
         return;
     }
 
+    std::string compiler_executable(void) const
+    {
+        return "@CMAKE_CXX_COMPILER@";
```

Comment:
In most cases, this likely wouldn't make a difference, but does this variable resolve to a full filepath?

Reply:
its the full file path. That wouldnt work out on the client machine if someone packaged the build with a non stock system compiler. On the other hand, its more likely to use the same compiler Pothos lib was built with

Reply:
Returning the basename() of the variable's path should resolve the issue.

Reply:
But isnt the fullpath probably the more robust implementation

Reply:
It may be a corner case, but I'm thinking about the case of a client having the compiler at a different path. This would allow the `PATH` to dictate which to use. `update-alternatives` could also take care of this, but it's still hardcoded on our end.